Tools/CISupport/ews-build/steps.py

```diff
@@ -59,9 +59,9 @@
 S3_BUCKET = f'ews-archives.webkit{custom_suffix}.org'
 S3_RESULTS_URL = f'https://ews-build{custom_suffix}.s3-us-west-2.amazonaws.com/'
 CURRENT_HOSTNAME = socket.gethostname().strip()
-EWS_BUILD_HOSTNAMES = ['ews-build.webkit.org', 'ews-build']
+EWS_BUILD_HOSTNAMES = load_password('EWS_BUILD_HOSTNAMES', default=['ews-build.webkit.org', 'ews-build'])
 TESTING_ENVIRONMENT_HOSTNAMES = ['ews-build.webkit-uat.org', 'ews-build-uat', 'ews-build.webkit-dev.org', 'ews-build-dev']
-EWS_URL = 'https://ews.webkit.org/'
+EWS_URL = load_password('EWS_BUILD_HOSTNAMES', default='https://ews.webkit.org/')
```

Comment:
`EWS_BUILD_HOSTNAMES` does seem accurate, it's not the hostname for buildbot (e.g: ews-build.webkit.org), it's the hostname for the django app (e.g.: ews.webkit.org). Plus you are already using this name above in line 62 for different purpose.

Can name it `EWS_APP_HOSTNAME`.

Reply:
@chgibb-apple did you miss this review comment?